cms/static/js/views/video_thumbnail.js

```diff
@@ -204,10 +256,17 @@ define(
 
             setActionInfo: function(action, showText, additionalSRText) {
                 this.$('.thumbnail-action').toggle(showText);
-                HtmlUtils.setHtml(
-                    this.$('.thumbnail-action .action-icon'),
-                    HtmlUtils.HTML(this.actionsInfo[action].icon)
-                );
+                if (action === 'error') {
+                    HtmlUtils.setHtml(
+                        this.$('.thumbnail-action .action-icon'),
+                        HtmlUtils.HTML('')
+                    );
```

Comment:
We are adding icon except when action is error. This is in regard to final UI mockups.